content/docs/examples/codelabs-tutorials.md

```diff
@@ -0,0 +1,42 @@
++++
+title = "Codelabs, Workshops, and Tutorials"
+description = "Below is a list of recommended end-to-end tutorials, workshops, walkthroughs,
```

Comment:
Suggestion, change the description to something like the following:

Recommended end-to-end tutorials, workshops, walk-throughs, and codelabs